src/app/utils/extractKnowledge/extractKnowledge.ts

```diff
@@ -0,0 +1,13 @@
+import { SupabaseResponse } from '@/types/SupabaseResponse';
+
+const extractKnowledge = (context: SupabaseResponse[]): string[] => {
+  return context.reduce((acc: string[], knowledge) => {
+    const inputText = knowledge.input_text;
+
+    acc.push(inputText);
+
+    return acc;
```

Comment:
what's acc?

Reply:
i usually use `acc` in `reduce` functions that output an array - means `accumulated` :)